numba/tests/test_array_methods.py

```diff
@@ -198,6 +198,12 @@ def check_err_size(arr, shape):
             self.assertEqual(str(raises.exception),
                              "total size of new array must be unchanged")
 
+        def check_err_negative(arr, shape):
```

Comment:
I'm only noticing now, but it seems confusing to have tests for reshape() in two different files (test_array_methods and test_array_manipulation). Perhaps we should consolidate them?